crawler.go

```diff
@@ -47,6 +47,9 @@ type Crawler struct {
 	// UserAgent specifies the user-agent for the remote server.
 	UserAgent string
 
+	// ForceAttemptHTTP2 specifies the corresponding field in the underlying HTTP transport.
+	ForceAttemptHTTP2 bool
```

Comment:
Instead of making this an option, I'd just remove the option and force it to `true` in line 202.
It seems that doing so is better behavior now (and consistent with `http.DefaultTransport`?)

Reply:
done